windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -936,9 +937,17 @@ function TCompileKeymanWeb.JavaScript_OutputString(FTabstops: string; fkp: PFILE
           end;
         CODE_DEADKEY:
           Result := Result + nlt + Format('k.KDO(%d,t,%d);', [len, recContext.Deadkey.Deadkey]);   // I4611
+        CODE_NOTANY:
+          begin
+            // #917: Minimum version required is 14.0: the KNO function was only added for 14.0
+            // Note that this is checked in compiler.cpp as well, so this error can probably never occur
+            if not IsKeyboardVersion14OrLater then
+              ReportError(fkp.Line, CERR_NotSupportedInKeymanWebContext, Format('Statement notany in context() match requires version 14.0+ of KeymanWeb', [GetCodeName(recContext.Code)]));  // I1971   // I4061
```

Comment:
Should the error inform the user to set `store(&version) '14.0'`?

Reply:
Missed this at the time, but given the error can't really occur I am keeping this lowpri